plugins/parsers/xml/testcases/cdc-vaccines.conf

```diff
@@ -0,0 +1,50 @@
+# Example for parsing COVID-19 Vaccine Distribution Allocations by Jurisdiction from CDC
+# 
+#
+# Expected Output:
+# cdc-vaccines,state=Connecticut,url=https://data.cdc.gov/api/views/saz5-9hgg/rows.xml,vaccine_type=Pfizer 1st_dose_allocations=60840i,2nd_dose_allocations=60840i 1617580800000000000
+# cdc-vaccines,state=Maine,url=https://data.cdc.gov/api/views/saz5-9hgg/rows.xml,vaccine_type=Pfizer 1st_dose_allocations=23400i,2nd_dose_allocations=23400i 1617580800000000000
+# cdc-vaccines,state=Massachusetts,url=https://data.cdc.gov/api/views/saz5-9hgg/rows.xml,vaccine_type=Pfizer 1st_dose_allocations=117000i,2nd_dose_allocations=117000i 1617580800000000000
+# cdc-vaccines,state=New\ York,url=https://data.cdc.gov/api/views/saz5-9hgg/rows.xml,vaccine_type=Pfizer 1st_dose_allocations=188370i,2nd_dose_allocations=188370i 1617580800000000000
+# cdc-vaccines,state=New\ York\ City,url=https://data.cdc.gov/api/views/saz5-9hgg/rows.xml,vaccine_type=Pfizer 1st_dose_allocations=143910i,2nd_dose_allocations=143910i 1617580800000000000
+
+
+[[inputs.http]]
+  ##  CDC data URLs from which to read XML metrics
+  urls = [
+    "https://data.cdc.gov/api/views/b7pe-5nws/rows.xml", # Moderna
+    "https://data.cdc.gov/api/views/saz5-9hgg/rows.xml", # Pfizer
+    "https://data.cdc.gov/api/views/w9zu-fywh/rows.xml" # Janssen/Johnson&Johnson
+    ]
+  data_format = "xml"
+  ## Drop hostname from list of tags
+  tagexclude = ["host"]
+
+    [[inputs.http.xml]]
+        metric_selection = "//row"
+        metric_name = "'cdc-vaccines'"
+        timestamp = "week_of_allocations"
+        timestamp_format = "2006-01-02T15:04:05"
+
+        [inputs.http.xml.tags]
+            state   = "jurisdiction"
+
+        [inputs.http.xml.fields_int]
+            1st_dose_allocations = "_1st_dose_allocations"
+            2nd_dose_allocations = "_2nd_dose_allocations"
```

Comment:
Not completely sure, but shouldn't this have the same indention level as `data_format` and `tagexclude`?